internal/backup/optimizations.go

```
@@ -376,23 +237,28 @@ func prefilterFiles(ctx context.Context, logger *logging.Logger, root string, ma
 			return nil
 		}
 
+		rel, relErr := filepath.Rel(root, path)
+		if relErr != nil {
+			return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 240) but it returns nil

(nilerr)